src/Arcus.Messaging.ServiceBus/Configuration/ServiceBusMessagePumpOptions.cs

```diff
@@ -19,13 +19,25 @@ public class ServiceBusMessagePumpOptions
         /// </summary>
         /// <remarks>The default value is 1</remarks>
         /// <exception cref="ArgumentException">Thrown when the <paramref name="value"/> is less than or equal to zero.</exception>
+        [Obsolete("Will be removed in v4, use " + nameof(MaxMessages) + " instead", DiagnosticId = "ARCUS")]
         public int MaxConcurrentCalls
         {
-            get => _maxConcurrentCalls;
+            get => MaxMessages;
+            set => MaxMessages = value;
+        }
+
+        /// <summary>
+        /// Gets the amount of messages that will be retrieved in one batch from Azure Service Bus.
+        /// </summary>
+        /// <remarks>The default value is 1</remarks>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="value"/> is less than or equal to zero.</exception>
+        public int MaxMessages
```

Comment:
I know I've coined `MaxMessages` but when I read the comments, maybe we should change it to `MaxMessagesPerBatch` or `MessagesPerBatch` ? wdyt ?

Reply:
thanks!